middle_end/flambda2/kinds/flambda_kind.ml

```diff
@@ -604,16 +624,23 @@ module With_subkind = struct
     | (Block _ | Float_block _), Any_value
     | Float_array, Any_value
     | Immediate_array, Any_value
+    | Value_array, Any_value
+    | Generic_array, Any_value
     | Boxed_float, Any_value
     | Boxed_int32, Any_value
     | Boxed_int64, Any_value
     | Boxed_nativeint, Any_value
-    | Tagged_immediate, Any_value ->
+    | Tagged_immediate, Any_value
```

Comment:
There could do with being a comment here since the one on line 622 has expired at this point.